internal/services/sentinel/sentinel_alert_rule_fusion_resource.go

```diff
@@ -62,6 +62,62 @@ func resourceSentinelAlertRuleFusion() *pluginsdk.Resource {
 				Optional: true,
 				Default:  true,
 			},
+
+			"source_setting": {
```

Comment:
setting is redundant?
```suggestion
			"source": {
```